features/openhab-demo-resources/src/main/resources/items/demo.items

```diff
@@ -100,7 +100,13 @@ Number Weather_Temp_Max         "Todays Maximum [%.1f °C]"      <temperature> (
 Number Weather_Temp_Min         "Todays Minimum [%.1f °C]"      <temperature> (Weather, Weather_Chart)
 Number Weather_Chart_Period     "Chart Period"
 DateTime Weather_LastUpdate     "Last Update [%1$ta %1$tR]" <clock>
-Number  Sun_Elevation           "Sun Elevation"             <sun>        (Weather)   { channel="astro:sun:home:position#elevation" }
+Number Sun_Elevation "Sun Elevation" <sun> { channel = "astro:sun:home:position#elevation" }
```

Comment:
Sorry, just noticed that all other lines have the labels and config strings aligned - it would [look nicer](https://github.com/mdilger/openhab-distro/blob/a80adeb48de4152917e648d13d93e9c1e1629cd0/features/openhab-demo-resources/src/main/resources/items/demo.items#L96), if you did the same here.